### ISSUE TITLE
Darstellung bei den Vermerken

### ISSUE THREAD

**Issue by FPfotenhauer** [SVWS-NRW/Schild3-BetaTest#1236]
Die Darstellung bei den Vermerken ist aus meiner Sicht ungünstig.
Hier sollte, wenn möglich, der gesamte text sichtbar sein.
Es ist auch bei sehr vielen Vermerken genug Platz vorhanden.

Es haben sich einige Schulen gewünscht, dass man das schneller aufrufen kann, da die Bemerkungen dort hin gewandert sind.
Da würde es schon helfen, wenn der Reiter direkt hinter "individauldaten II" wäre.

Können wir das evtl. einmal in einer nächsten Version ausprobieren? Wenn das keinen Aufwand erzeugt natürlich.

**Comment by JuergenRichter** [comment]
Macht keinen Aufwand. Man kann aber meines Wissens auch die Zeilenhöhe bei den Vermerken ändern, das kann man dann so einstellen, dass man mehr Text der Bemerkung sieht.

**Comment by FPfotenhauer** [comment]
Kann ich das als User machen?

**Comment by FPfotenhauer** [comment]
So ist das im Webclient umgesetzt. Es geht hier nur darum, dass die Lesbarkeit etwas besser ist.

![Bildschirmfoto vom 2024-12-18 13-48-43](https://github.com/user-attachments/assets/07c42126-d235-4329-911f-59aac21c2c72)

**Comment by JuergenRichter** [comment]
> Kann ich das als User machen?
Ja, mit der Maus am linken Rand auf den Trennstrich "fahren", dann sollte da ein entsprechender  Cursor erscheinen
![image](https://github.com/user-attachments/assets/19e688b6-01a9-41d4-b473-f561cdaf998b)
Wir können das aber auch durch ein kleines Plus-Zeichen umsetzen (wie z.B. bei den Fächern, um die Teilleistungen aufzuklappen)

**Comment by JuergenRichter** [comment]
Ich habe die Position des Reiters jetzt mal hinter "Erziehungsberechtigte/Telefonnummern" gesetzt und auch die Option geschaffen, den Bemerkungstext aufzuklappen.
